client/components/form.js

```diff
@@ -2,22 +2,19 @@ const OForms = require('o-forms');
 import { sendMessage, dispatchTrackingEvent } from './utils';
 
 const overlay = document.querySelector('.prospect-form__overlay');
-const errorMessage = document.querySelector('.prospect-form__message');
 
 const notify = (form) => {
   sendMessage({
-    height: form.clientHeight + 5
+    height: form.clientHeight + 20
```

Comment:
What's this magic number for? Could be worth adding a comment maybe.

Reply:
Just a buffer to allow for paddings etc